Jenkinsfile

```diff
@@ -1,6 +1,13 @@
+// Jenkinsfile: scripted Jenkins pipefile
+// [revision: Jenkinsfile@20191122-02]
+// This file was automatically generated using 'python -m vsc.install.ci -f'
+// DO NOT EDIT MANUALLY
+
 node {
-  stage 'checkout git'
-  checkout scm
-  stage 'test'
-  sh "python setup.py test"
-}
+    stage 'checkout git'
```

Comment:
Note to self: use new syntax here (see https://github.com/hpcugent/quattor/blob/master/Jenkinsfile)

(thx @wdpypere)

Reply:
fixed in cdd0740